tests/cypress/e2e/helpers.ts

```diff
@@ -3,8 +3,8 @@ export const inputGraphTypeSelector = 'input[id="input_Graph type"]';
 export const inputHostLabelsSelector = 'input[id="react-select-15-input"]';
 export const inputHostSelector = 'input[id="input_Hostname"]';
 export const inputMetricSelector = 'input[id="input_Metric"]';
+export const inputRegexSelector = 'input[class="css-1mlczho-input-input"]'; //TODO: introduce id for this input selector.
```

Comment:
i think 893bdffd45371431b153998582d8fd8d38eb295a should fix this and give you two ids for the two different regex fields.

Reply:
Nice!
I can wait for it to be merged and then rebase.

Reply:
Done.